cypress/integration/common/settings.js

```diff
@@ -121,3 +121,34 @@ Then('it gets deleted successfully', () => {
   cy.get('.iziToast-message')
     .should('contain', 'Deleted social media')
 })
+
+When('I start editing a social media link', () => {
+  cy.get("a[name='edit']")
+    .click()
+})
+
+Then('I can cancel editing', () => {
+  cy.get('button#cancel')
+    .click()
+    .get('input#editSocialMedia')
+    .should('have.length', 0)
+})
+
+When('I edit and save the link', () => {
+  cy.get('input#editSocialMedia')
+    .clear()
+    .type('https://freeradical.zone/tinkerbell')
+    .get('button')
+    .contains('Save')
+    .click()
+})
+
+Then('the new url is displayed', () => {
+  cy.get("a[href='https://freeradical.zone/tinkerbell']")
+    .should('have.length', 1)
+})
+
+Then('the old url is not displayed', () => {
+  cy.get("a[href='https://freeradical.zone/peter-pan']")
+    .should('have.length', 0)
+})
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T07:24:04Z" title="Thursday, July 18th 2019, 9:24:04 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -121,3 +121,34 @@ Then('it gets deleted successfully', () => {
   cy.get('.iziToast-message')
     .should('contain', 'Deleted social media')
 })
+
+When('I start editing a social media link', () => {
+  cy.get("a[name='edit']")
+    .click()
+})
+
+Then('I can cancel editing', () => {
+  cy.get('button#cancel')
+    .click()
+    .get('input#editSocialMedia')
+    .should('have.length', 0)
+})
+
+When('I edit and save the link', () => {
+  cy.get('input#editSocialMedia')
+    .clear()
+    .type('https://freeradical.zone/tinkerbell')
+    .get('button')
+    .contains('Save')
+    .click()
+})
+
+Then('the new url is displayed', () => {
+  cy.get("a[href='https://freeradical.zone/tinkerbell']")
+    .should('have.length', 1)
+})
+
+Then('the old url is not displayed', () => {
+  cy.get("a[href='https://freeradical.zone/peter-pan']")
+    .should('have.length', 0)
+})
```


Wonderful done @alina-beck !!! 👍🏼
Love it! 😻